mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -1338,7 +1339,7 @@ impl TakerSwap {
             ]));
         }
 
-        let timeout = self.r().data.started_at + self.r().data.lock_duration / 3;
+        let timeout = self.r().data.maker_payment_wait;
```

Comment:
Before we waited for
```rust
let timeout = self.r().data.started_at + self.r().data.lock_duration / 3;
```

But now we have `maker_payment_wait`:
https://github.com/KomodoPlatform/atomicDEX-API/blob/c4266a6f82edf8098fe42532e1e3c554cb220f1d/mm2src/mm2_main/src/lp_swap/taker_swap.rs#L985
https://github.com/KomodoPlatform/atomicDEX-API/blob/c4266a6f82edf8098fe42532e1e3c554cb220f1d/mm2src/mm2_main/src/lp_swap.rs#L347-L349

Is this expected?

Reply:
Yes, this was a really minor point but while validating the maker payment, the `validate_maker_payment` method was taking as input the `try_spv_proof_until: self.r().data.maker_payment_wait` and that is `swap_started_at + (locktime * 2) / 5`: 

https://github.com/KomodoPlatform/atomicDEX-API/blob/c4266a6f82edf8098fe42532e1e3c554cb220f1d/mm2src/mm2_main/src/lp_swap/taker_swap.rs#L1283-L1295

And right after that in the `send_taker_payment ` method there was this condition which stops the swap if now >  `self.r().data.started_at + self.r().data.lock_duration / 3`, which is less than `maker_payment_wait `: 

https://github.com/KomodoPlatform/atomicDEX-API/blob/1f30ec7ecaebe8f2524b851c5656844eaf3a8237/mm2src/mm2_main/src/lp_swap/taker_swap.rs#L1341-L1347

I thought this is inconsistent and either one of them should be made equal to the other one.

Reply:
Thank you for the clarification! Makes sense!